### PR TITLE
made social share buttons on PDP ADA compliant

### PR DESCRIPTION
**Pull request by minimart**
Solves Issue #2182 

Aria (and other screen-readers) will now state that each of the social buttons, if clicked, will "Share on {SocialNetworkName}".

#### Testing

Using [aXe](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) or any other ADA compliance checker, review the warnings present on the PDP. You could also utilize a screen reader to hear the text being read to the user. 

_Note: There are a number of other complaints from this tool, some of which I would like to keep making Issues and PRs for to make sure that Reaction Commerce is, on its base, ADA compliant._

### CODE REVIEW

File: imports/plugins/included/social/client/components/facebook.js [hunk 1]
```diff
@@ -88,7 +88,9 @@ class FacebookSocialButton extends Component {
     });
 
     return (
-      <a className="btn btn-flat facebook-share" href="#" onClick={this.handleClick} target="_blank">
+      <a className="btn btn-flat facebook-share" aria-label="Share to Facebook" href="#" onClick={this.handleClick}
```
Comment (aaronjudd) [inline imports/plugins/included/social/client/components/facebook.js]: Perhaps the aria labels also be i18n translated?

Comment (minimart) [inline imports/plugins/included/social/client/components/facebook.js]: Good call! I'll make a separate namespace for all of the aria translations as well. 👍

Comment (Pheonix7284) [inline imports/plugins/included/social/client/components/facebook.js]: Generally if you are using a href="#" the a tag should probably be a button tag as you aren't 'linking' or 'anchoring' to another part of the page!

**Comment by aaronjudd** [comment]
@minimart both suggestions are good ones.. I can accept this PR right now though as it certainly causes no harm.

**Comment by minimart** [comment]
Sounds good! This is working for now-- and I can keep working on it and either keep this PR open and update it or open another one-- whichever works better for you!

**Pull request merged**